blob/gcsblob/gcsblob_test.go

```diff
@@ -87,7 +87,7 @@ func newHarness(ctx context.Context, t *testing.T) (drivertest.Harness, error) {
 }
 
 func (h *harness) HTTPClient() *http.Client {
-	return &h.client.Client
+	return &http.Client{Transport: h.rt}
```

Comment:
Note: this was a bug previously -- `HTTPClient` is supposed to return an unauthorized HTTP client, but it was returning an authorized one. Fixed by updating `setup.go` (below) to return the `http.RoundTripper` transport before adding the authorization wrapper, and creating a new HTTPClient using that transport here.